common/lib/xmodule/xmodule/tabs.py

```diff
@@ -283,7 +283,7 @@ def fragment_view(self):
         Returns the view that will be used to render the fragment.
         """
         if not self._fragment_view:
-            self._fragment_view = get_storage_class(self.fragment_view_name)()
+            self._fragment_view = import_string(self.fragment_view_name)()
```

Comment:
get_storage_class does the same thing, but import_string more clearly maps to what we'd like to do here.